Formula/iam-policy-json-to-terraform.rb

```diff
@@ -0,0 +1,28 @@
+class IamPolicyJsonToTerraform < Formula
+  desc "Convert a JSON IAM Policy into terraform"
+  homepage "https://github.com/flosell/iam-policy-json-to-terraform"
+  url "https://github.com/flosell/iam-policy-json-to-terraform/archive/1.3.0.tar.gz"
+  sha256 "90b5be7caa232186718b5645b2d5ab65a59695854db7598462bdbe059444a051"
+  head "https://github.com/flosell/iam-policy-json-to-terraform.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    dir = buildpath/"src/github.com/flosell/iam-policy-json-to-terraform"
+    dir.install buildpath.children
+    cd "src/github.com/flosell/iam-policy-json-to-terraform" do
+      system "make", "vendor", "tools"
+      system "make", "iam-policy-json-to-terraform_darwin"
+      mv "iam-policy-json-to-terraform_darwin", "iam-policy-json-to-terraform"
+      bin.install "iam-policy-json-to-terraform"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system bin/"iam-policy-json-to-terraform", "-version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks for reviewing. I added a more detailed input and output check to verify cli functionality.